src/explorer/WebAppTreeItem.ts

```diff
@@ -46,9 +47,9 @@ export class WebAppTreeItem extends SiteTreeItem {
     }
 
     public compareChildrenImpl(ti1: AzureTreeItem<ISiteTreeRoot>, ti2: AzureTreeItem<ISiteTreeRoot>): number {
```

Comment:
You should probably move `compareChildrenImpl` to `SiteTreeItem` since `WebJobsNATreeItem` could also show up under a slot